mainnet.toml

```diff
@@ -1,118 +1,118 @@
 'v1-000000-000100-beaconblocks.seg' = 'eaee23c3db187c8be69e332b4ff50aa73380d0ef'
-'v1-000000-000500-bodies.seg' = 'e9b5c5d1885ee3c6ab6005919e511e1e04c7e34e'
-'v1-000000-000500-headers.seg' = 'df09957d8a28af3bc5137478885a8003677ca878'
+'v1-000000-000500-bodies.seg' = 'e9b5c5d1885ee3c6ab6005919e511e1e04c7e34e'      
```

Comment:
something wrong with your terminal settings. probably you do copy it from the screen and it has some spaces in end of lines